packages/migrate/src/utils/test-handlePanic.ts

```diff
@@ -11,9 +11,16 @@ async function main() {
   )
 
   const packageJsonVersion = '0.0.0'
-  const engineVersion = '734ab53bd8e2cadf18b8b71cb53bf2d2bed46517'
+  const enginesVersion = '734ab53bd8e2cadf18b8b71cb53bf2d2bed46517'
+  const getDatabaseVersionSafe = () => Promise.resolve(undefined)
```

Comment:
Could you use the one from `getDatabaseVersionSafe.ts` ?

Reply:
I see that the same test in `packages/cli` uses the literal `prismaEngineVersionHash` (https://github.com/prisma/prisma/blob/dba8c89d305d91b7cdd6cac6fea49f5192d39af3/packages/cli/src/utils/test-handlePanic.ts#L8), I may just use the same in `packages/migrate`

Reply:
Actually this file is not even used anywhere, it's a small script that someone can run locally (not part of tests)

So anything works I guess, but let's still remove `const getDatabaseVersionSafe = () => Promise.resolve(undefined)` so we don't forget later

Reply:
Why would we remove `const getDatabaseVersionSafe = () => Promise.resolve(undefined)`? It's purposedly a mocked function